src/pages/Dashboard/Settings/WidgetSettings/WidgetSettings.jsx

```diff
@@ -152,26 +152,41 @@ const WidgetSettings = () => {
       const searchEventsFilters = encodeURIComponent(filtersParam);
 
       const locale = onLanguageSelect(allValues?.language);
-      const urlCopy = new URL('https://s3.ca-central-1.amazonaws.com/staging.cms-widget.footlight.io/index.html');
+      const urlCopy = new URL(widgetUrl);
+      const urlCopyMobile = new URL(widgetUrl);
```

Comment:
We don't need to define urlCopy and urlCopyMobile using new URL. Instead, we can directly assign urlCopy to urlCopyMobile.

Reply:
if we use 
const urlCopy = new URL(widgetUrl);
const urlCopyMobile = urlCopy;

to define them, then each change made to urlCopy will be applied on urlCopyMobile as they share the same reference.

We need both of them to be independant.